src/shared/components/community/community.tsx

```diff
@@ -712,6 +715,16 @@ export class Community extends Component<CommunityRouteProps, State> {
             />
           </div>
         )}
+        <div className="col-auto ps-0">
+          <button
+            class="btn btn-secondary"
+            onClick={() => {
+              this.setState({ expandAllImages: !this.state.expandAllImages });
+            }}
+          >
+            <Icon icon={this.state.expandAllImages ? "minus" : "plus"} />
+          </button>
```

Comment:
Visually showing only the icon is fine, but there should be some visually hidden text for screen reader users. You can add a span as a child of the button and hide it using the [`visually-hidden` Bootstrap class](https://getbootstrap.com/docs/5.2/getting-started/accessibility/#visually-hidden-content).

[This accessibility section](https://www.accessibility-developer-guide.com/knowledge/aria/bad-practices/#adding-missing-labels) explains the benefits of this approach over using `aria-label`.

Reply:
Makes sense. I did it in a different way, by adding a prop `altText` for `Icon`. That one seems to handle accesibility in a different way, but it can be fixed in another PR.

Reply:
See comment above, use aria labels on the buttons like we do for every icon button. 

I disagree with that article, [it seems to never want you to use aria-labels](https://www.accessibility-developer-guide.com/knowledge/aria/bad-practices/#approach-using-aria) ever.

Just follow the standard we use for every icon button already in lemmy-ui instead of trying to recreate from scratch.

Use `aria-label` and `data-tippy-content` with i18n strings. Check out `src/shared/components/common/vote-buttons.tsx` for an example.

Reply:
Updated